### PR TITLE
New version: PrismLauncher.PrismLauncher version 8.0

### PR DESCRIPTION
**Pull request by PrismAutomata**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/125053)

### CODE REVIEW

**Review by russellbanks — APPROVED** [review]

File: manifests/p/PrismLauncher/PrismLauncher/8.0/PrismLauncher.PrismLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-11-07
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/8.0/PrismLauncher-Windows-MSVC-Setup-8.0.exe
+  InstallerSha256: 84896FBFB13875AC47D85739E4B55E34F0F60A183C27077426CF839020D91E13
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Comment (Scrumplex) [inline manifests/p/PrismLauncher/PrismLauncher/8.0/PrismLauncher.PrismLauncher.installer.yaml:19]: This isn't valid. There are no x86 builds anymore

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---

File: manifests/p/PrismLauncher/PrismLauncher/8.0/PrismLauncher.PrismLauncher.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/p/PrismLauncher/PrismLauncher/8.0/PrismLauncher.PrismLauncher.installer.yaml:19]:
```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/8.0/PrismLauncher-Windows-MSVC-Setup-8.0.exe
  InstallerSha256: 84896FBFB13875AC47D85739E4B55E34F0F60A183C27077426CF839020D91E13
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
Hi @PrismAutomata,

It appears that one of the installers isn't valid.

**Review by txtsd — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by txtsd** [comment]
Just a heads-up: there are unresolved conversations.

**Pull request merged**